packages/social-previews/test/index.js

```diff
@@ -0,0 +1,287 @@
+/**
+ * Internal dependencies
+ */
+import React from 'react';
+import {
+	FacebookPreview as Facebook,
+	TwitterPreview as Twitter,
+	SearchPreview as Search,
+} from '../src';
+import { shallow } from 'enzyme';
+
+const IMAGE_SRC_FIXTURE = 'https://wordpress.com/someimagehere';
+
+describe( 'Facebook previews', () => {
+	it( 'should expose a Facebook preview component', () => {
+		expect( Facebook ).not.toBe( undefined );
+	} );
+
+	it( 'should display a (hard) truncated title', () => {
+		const wrapper = shallow(
+			<Facebook title="I am the very model of a modern Major-General, I've information vegetable, animal, and mineral." />
+		);
+
+		const titleEl = wrapper.find( '.facebook-preview__title' );
```

Comment:
I would advice adding a `data-testid` attribute to all elements to be tested as per 
https://kentcdodds.com/blog/making-your-ui-tests-resilient-to-change

Also, it would be great (separate PR?) to add a babel script that removes them from production builds
https://www.npmjs.com/package/babel-plugin-jsx-remove-data-test-id

Reply:
In general I"m not a fan of Enzyme at all. I feel it promotes suboptimal testing practices and makes writing bad tests too easy.

If I had it my way I would use RTL or even just "pure" React Testing Utils to write these tests.

My ideal practice is asserting against elements based on what the user can perceive. Principally text content but falling bad to semantic `roles` or `aria` if necessary. 

I don't mind the `testid` approach but I feel it should be a last resort. Ideally, I'd prefer to spend time rewriting the tests to select based on text or even find a way to use React Testing Library.

For now however, for the sake of expediency, I've just followed the existing paths laid out in other packages.